### DISCUSSION TITLE
Macintosh tag not updated in shared folder

### DISCUSSION THREAD

**Discussion by PaulDeCarli** [samschott/maestral#486] [Q&A]
I've sent a file to a shared folder. The recipient tagged the file (green) but the tag does not update in my version of the shared folder.
This is something that worked when we both used the Dropbox clients.
Is this a known limitation?

**Comment by samschott** [comment]
Yes, it is. Metadata such as file tags are not currently exposed by the public Dropbox API - I'm afraid you'll need to use the official client for that.
